lib/resources/example_json/r4_examples_allergy_intolerance.rb

```diff
@@ -211,5 +219,53 @@ module Resources
         }
       ]
     }
+
+    R4_ALLERGY_INTOLERANCE_UPDATE ||= {
+      "resourceType": "AllergyIntolerance",
+      "id": "7403733",
+      "clinicalStatus": {
+          "coding": [
+              {
+                  "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
+                  "code": "active"
+              }
+          ]
+      },
+      "verificationStatus": {
+          "coding": [
+              {
+                  "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-verification",
+                  "code": "unconfirmed"
+              }
+          ]
+      },
+      "category": [
+          "food"
+      ],
+      "code": {
+          "coding": [
+              {
+                  "system": "http://snomed.info/sct",
+                  "code": "91935009",
+                  "display": "Allergy to peanuts (disorder)"
+              }
+          ],
+          "text": "Allergy to peanuts"
+      },
+      "patient": {
+          "reference": "Patient/4342008",
+          "display": "SMART, WILMA"
+      },
+      "recordedDate": "2019-02-08T00:00:00Z",
+      "recorder": {
+          "reference": "Practitioner/2042017",
+          "display": "Generated Domain User for 0"
+      },
+      "note": [
+        {
+          "text": "Updated with additional details provided by patient."
```

Comment:
Could we update this field to better reflect notes with ID's and notes without?

Reply:
Added an existing note field, do you think the note text is too on the nose? https://github.com/cerner/fhir.cerner.com/pull/327/commits/31f33cdc9cac89a6e5fc99c1a6d77e2683f8d674

Reply:
To me it's perfectly where it should be. But, i am wondering about out strategy on if we want to keep our example body's understandable or medically accurate. 

Thoughts?

Reply:
After discussion offline we decided to go with more medically accurate note descriptions and then add extra documentation detailing how note works. https://github.com/cerner/fhir.cerner.com/pull/327/commits/57930130970b544cc9a59acfc45e6ad0583c87fb @praneethk2094  please review this commit.